juniper_official/Linecard/pipe-stats.rule

```diff
@@ -0,0 +1,224 @@
+healthbot {
```

Comment:
Address in this rule all comments mentioned in above rule 41.

Reply:
Done